extensions/sql-database-projects/README.md

```diff
@@ -22,6 +31,19 @@ Please report issues and feature requests [here.](https://github.com/microsoft/a
 * Start from an existing database by using the `Create Project from Database` from the command palette or database context menu.
```

Comment:
Remove first "the" in the sentence, fixed to:  Start from an existing database by using `Create Project from Database` from the command palette or database context menu.